From 5cefddeaeea8f75d12e2b04d37e9302803008af3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Piotr=20Dr=C4=85g?= <piotrdrag@gmail.com>
Date: Sat, 28 Jan 2017 23:52:32 +0100
Subject: [PATCH 1/2] Remove X-GNOME-FullName from the .desktop file

It's a GNOME-specific hack for the 2.x era menu system that was never
standardized in [1] and is now unused in GNOME 3.

[1] https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html
---
 gtk/transmission-gtk.desktop.in | 1 -
 1 file changed, 1 deletion(-)

diff --git a/gtk/transmission-gtk.desktop.in b/gtk/transmission-gtk.desktop.in
index 51ee8b771..2b6e799b8 100644
--- a/gtk/transmission-gtk.desktop.in
+++ b/gtk/transmission-gtk.desktop.in
@@ -1,7 +1,6 @@
 [Desktop Entry]
 _Name=Transmission
 _GenericName=BitTorrent Client
-_X-GNOME-FullName=Transmission BitTorrent Client
 _Comment=Download and share files over BitTorrent
 Exec=transmission-gtk %U
 Icon=transmission
-- 
2.14.3

